App/app/src/main/res/values/colors.xml

```diff
@@ -10,4 +10,5 @@
     <color name="BackgroundAccent">#FFC5C5</color>
     <color name="BackgroundDark">#CC9292</color>
     <color name="LinkBase">#2196F3</color>
+>>>>>>> develop
```

Comment:
@Glordir @IrenaRuprecht is that an artifact from the merge?
Should it be removed?